### PR TITLE
Adding feature to specify a custom hipchat server

### PR DESCRIPTION
**Pull request by vmule**
Some people do use a custom hipchat server.
Making this possible to use.
Cheers

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -5,6 +5,7 @@
 class puppet_hipchat (
   $api_key,
   $room,
+  $server         = 'https://api.hipchat.com'
```
Comment (jamtur01) [inline manifests/init.pp]: You're missing a comma here - that's why the tests fail.

Comment (vmule) [inline manifests/init.pp]: Sorry, stupid mistake.
Comma added.
Thanks

**Comment by jamtur01** [comment]
Tests still failing?

**Comment by vmule** [comment]
Yes but looks like unrelated to my changes if I'm not wrong..

**Comment by jamtur01** [comment]
Okay - tests fixed! Thanks for your contribution!

**Pull request merged**